### DISCUSSION TITLE
Is there a way to make the content textarea wider ?

### DISCUSSION THREAD

**Discussion by spapas** [django-wiki/django-wiki#1344] [Q&A]
Right now it is too narrow making editing difficult.

**Comment by benjaoming** [comment]
Do you mean that if the sidebar could be hidden, there'd be more space?

![image](https://github.com/django-wiki/django-wiki/assets/374612/23d90c0e-25bd-4a32-94cc-3b0d30ca8a20)

**Reply by spapas** [reply]
Something like that would be great yes... Or change the form to have the labels on top instead of to the left?
